client/src/api/product-api.js

```diff
@@ -0,0 +1,33 @@
+import request from './index';
+
+export const getRelatedIds = () => {
+  return request.get(
+    'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/37311/related',
+  );
+};
+
+export const getDetailById = (id) => {
+  return request.get(
+    `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/products/${id}/styles`,
+  );
+};
+
+export const getReviewById = (id) => {
+  return request.get(
+    'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfe/reviews/',
```

Comment:
I think you're using this to get the average review score for each product. If so, you should be using the `/reviews/meta` endpoint to get a count of all reviews by star number.